am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -122,4 +126,36 @@ public FilterResourceResponse filterResource(String userId, String resourceId, J
 
         return null;
     }
+
+
+    /**
+     * <p>
```

Comment:
I do not think there is point starting with <p>. Haven't seen any Javadoc doing it. Suggest removing it.

Reply:
👍